dracut.sh

```diff
@@ -2467,29 +2464,43 @@ if [[ $uefi == yes ]]; then
         fi
     fi
 
+    offset=$(objdump -h "$uefi_stub" | awk 'NF==7 {size=strtonum("0x"$3);\
```

Comment:
Just a minor nitpick, I'd save the output of `objdump -h "$uefi_stub"` to an intermediate variable to check its return code and exit with a dfatal message if it's non-zero.

Reply:
Thank you for your feedback.
I add a protection on the result of the command, to be sure that an offset is really taken.